README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
-| Test suite execution timing                                                               | ✔️       | ✔️                                                           | ✖️                                                                |
-| Helpful output to find slow tests                                                         | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | C
+| Test suite execution timing                                                               | ✔️       | ✔️                                                           | ✔️                                                                |
+| Helpful output to find slow tests                                                         | ✔️       | [Lib](http://hackage.haskell.org/package/hspec-slow)        | C                                                                |
 | Coloured output                                                                           | ✔️       | ✔️                                                           | ✔️                                                                |
 | Colourless output                                                                         | ✔️       | ✔️                                                           | ✔️                                                                |
-| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | ✖️                                                                |
-| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | ✔️                                                                |
-| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | ✖️                                                                |
+| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | C                                                                |
+| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hunit)           |
+| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | C                                                                |
```

Comment:
I changed this to a `C` because I'm guessing that you can get tasty to output unicode trees?

Reply:
I don't think you can. If it's really about the test tree itself, this is probably X.